src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -391,6 +396,7 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
             responseId: '',
           });
         });
+        this.isFeedbackSessionQuestionResponsesLoading = false;
```

Comment:
You should set `isFeedbackSessionQuestionResponsesLoading` to `true` somewhere?

Reply:
It's initialised as `true`

Reply:
We should not depends on the initial values of component. `true` need to set properly when the loading actually begins.